openapi_python_client/templates/property_templates/union_property.py.jinja

```diff
@@ -66,10 +66,12 @@ elif {{ source }} is None:
     {% else %}
         {% set ns.contains_modified_properties = true %}
     {% endif %}
-    {% if loop.first and property.required and not property.nullable %}{# No if UNSET or if None statement before this #}
+    {% if not ns.has_if %}
 if isinstance({{ source }}, {{ inner_property.get_instance_type_string() }}):
+        {% set ns.has_if = true %}
     {% elif not loop.last or ns.contains_properties_without_transform %}
 elif isinstance({{ source }}, {{ inner_property.get_instance_type_string() }}):
+    {% set has_first_clause = true %}
```

Comment:
👀

Reply:
Ahhh! What's that doing there? Nothing I think 😅